tests/png-suite/tests/transparency.t/run.t

```diff
@@ -0,0 +1,30 @@
+  $ ../test.sh *.png
+  Testing tbbn0g04.png
+  Corrupted image: cannot write greyscale image with alpha to PNG with bitdepth of 4.
```

Comment:
This error happens when writing the output PNG, and has to do with the fact that we need to emit a `tRNS` chunk for this particular image. We should fix this, but not in this PR.